browser/widevine/widevine_utils.cc

```diff
@@ -121,9 +112,89 @@ void InstallBundleOrRestartBrowser() {
                                    true);
   }
 }
+
+void SetWidevineInstalledVersion(const std::string& version) {
+  g_browser_process->local_state()->SetString(kWidevineInstalledVersion,
+                                              version);
+}
+
+std::string GetWidevineInstalledVersion() {
+  return g_browser_process->local_state()->GetString(kWidevineInstalledVersion);
+}
 #endif
 
+int GetWidevinePermissionRequestTextFrangmentResourceId() {
+  int message_id = -1;
+
+#if BUILDFLAG(ENABLE_WIDEVINE_CDM_COMPONENT)
```

Comment:
just like `BUNDLE_WIDEVINE_CDM`, `ENABLE_WIDEVINE_CDM_COMPONENT` only serves to make our code more confusing. If widevine is enabled then it is enabled as a component. There is no other option for us so all we care about is whether widevine is enabled or not